test/update-manifest-from-html-test.js

```diff
@@ -0,0 +1,134 @@
+'use strict';
+const { expect } = require('chai');
+const Project = require('fixturify-project');
+const fixturify = require('fixturify');
+const fs = require('fs');
+const updateManifestFromHtml = require('../lib/embroider/update-manifest-from-html')
+
+describe('embroider/update-manifest-from-html', function() {
+  const PROJECT = new Project('my-app');
+
+  PROJECT.files['index.html'] = `
+  <script src="a.js"></script>
+  <script src="b.js"></script>
+  <script src="%PLACEHOLDER[c.js]%"></script>
+  <script src="/some_context_path/e.js"></script>
+  <script src="chunk-12.js"></script>
+  <script src="chunk-12.test.js"></script>
+  <script src="fastboot-ignore.js" data-fastboot-ignore></script>
+  <script src="ember-cli-live-reload.js"></script>
+  <script src="testem.js"></script>
+  <script src="assets/vendor.js"></script>
+  <script src="assets/vendor-static.js"></script>
+  `;
+
+  PROJECT.pkg.fastboot = {
+    manifest: {
+      htmlFile: 'index.html',
+      appFiles: ['assets/app.js', 'assets/my-app-fastboot.js'],
+      vendorFiles:['assets/vendor.js', 'assets/vendor-static.js']
+    }
+  };
+  const project = PROJECT.clone();
+  project.writeSync();
+  const files = {};
+  files['a.js'] = '';
+  files['b.js'] = '';
+  files['c.js'] = '';
+  files['d.js'] = '';
+  files['e.js'] = '';
+  files['chunk-12.js'] = '';
+  files['chunk-12.test.js'] = '';
+
+  it(`works correctly as files are added/removed`,  function() {
```

Comment:
I didn't see a test case for `data-fastboot-ignore` being tested..

Reply:
https://github.com/ember-fastboot/ember-cli-fastboot/blob/9ace0cd548a155d2ac92616ca0eb9acb97088423/test/update-manifest-from-html-test.js#L18

Reply:
https://github.com/ember-fastboot/ember-cli-fastboot/blob/29aefe0b8dfa8af62cb22fea671386a3f085e592/test/update-manifest-from-html-test.js#L63-L69